StRoot/StSpinPool/StFcsEpdQaMaker/StFcsEpdQaMaker.cxx

```diff
@@ -38,26 +38,37 @@ Int_t StFcsEpdQaMaker::Init(){
     return kStFatal;
   }
   
-  if(mFilename[0]==0 && mRun>0){
+  if(mFilename[0]=='0' && mRun>0){
       int yday=mRun/1000;
       sprintf(mFilename,"%d/%d.epdqa.root",yday,mRun);
       printf("StFcsEpdQaMaker::Init - Opening %s\n",mFilename);
   }
   mFile=new TFile(mFilename,"RECREATE");
-
-  char t[100], tt[100];
-  for(int i=0; i<16; i++){
-    sprintf(t,"QtDepAdcCh%d",i);
-    sprintf(tt,"Dep01Ch%d-PP10TT%d; QTADC; DEP Fit Integral",i,i*2);
-    mQtDepA[i] = new TH2F(t,tt,256,0,1024,256,0,1024*4);
-    sprintf(t,"QtDepTacCh%d",i);
-    sprintf(tt,"Dep01Ch%d-PP10TT%d; QTTAC; DEP Peak Timebin",i,i*2);
-    mQtDepT[i] = new TH2F(t,tt,100,0,3000,100,45,56);
-    sprintf(t,"QtDepRatCh%d",i);
-    sprintf(tt,"Dep01Ch%d-PP10TT%d; DEP Peak Timebin; QTADC/DEPIntg;",i,i*2);
-    mQtDepR[i] = new TH2F(t,tt,100,44,57,100,0.0,0.8);
-  }
   
+  char t[100], n[100];
```

Comment:
I realize this is pool code and won't be used in production... but any reason why we are avoiding the standard logging facility?